Lib9c/Action/RankingBattle.cs

```diff
@@ -80,8 +80,18 @@ public override IAccountStateDelta Execute(IActionContext context)
 
             sw.Stop();
             Log.Verbose("{AddressesHex}RankingBattle Get AgentAvatarStates: {Elapsed}", addressesHex, sw.Elapsed);
+
             sw.Restart();
+            var sheets = states.GetSheets(
+                containRankingSimulatorSheets:true,
+                sheetTypes: new[]
+                {
+                    typeof(CostumeStatSheet),
+                });
```

Comment:
```suggestion
                sheetTypes: new[]
                {
                    typeof(CharacterSheet),
                    typeof(CostumeStatSheet),
                });
```

There is `sheets.GetSheet<CharacterSheet>()` below so how about you think to add a type of `CharacterSheet` here?